.travis.yml

```diff
@@ -117,6 +117,10 @@ script:
   - rm -f kubeseal-static
   - GOOS=windows GOARCH=amd64 make kubeseal-static
   - cp kubeseal-static kubeseal.exe
+  - GOOS=linux GOARCH=arm GOARM=7 make kubeseal-static
+  - cp kubeseal-static kubeseal-arm
+  - GOOS=linux GOARCH=arm GOARM=8 make kubeseal-static
```

Comment:
According to https://github.com/golang/go/wiki/GoArm arm8 is not enabled by setting GOARM=8 but by GOARCH=arm64

Could you please double-check?